docker/docker/src/test/java/org/arquillian/cube/docker/impl/client/CubeConfigurationTest.java

```diff
@@ -25,6 +28,7 @@
 import org.hamcrest.collection.IsIterableContainingInOrder;
 import static org.junit.Assert.assertThat;
 
+@Category({RequiresDocker.class, RequiresDockerMachine.class})
```

Comment:
This looks like a unit test to me - why do we require docker category here?

Reply:
same like above